packages/editor/src/dataviews/store/private-selectors.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Internal dependencies
+ */
+import type { State } from './reducer';
+
+export function getEntityActions( state: State, kind: string, name: string ) {
+	return state.actions[ kind ]?.[ name ] ?? [];
```

Comment:
```
return state.actions[ kind ]?.[ name ] ?? EMPTY_ARRAY;
```

Let's return a stable fallback reference.

Fixes the "The 'useSelect' hook returns different values when called with the same state and parameters. This can lead to unnecessary rerenders. " warning.

Reply:
https://github.com/WordPress/gutenberg/pull/62536